cmake/emu/armfvp.cmake

```diff
@@ -10,11 +10,24 @@ find_program(
   NAMES ${ARMFVP_BIN_NAME}
   )
 
+if(CONFIG_ARMV8_A_NS)
```

Comment:
Commit message says:
> It may be needed to run Zephyr at EL1NS level with `CONFIG_ARMV8_A_NS`
> In order to run at EL1NS, you'll need a proper Firmware loaded in the
FVP model to run Zephyr at non-secure EL3.

but based on this code it appears to be required to set those environment variables or things will break.

I think there is a missing check that the required environment variables are set and points to valid files.

Reply:
OK, I'll try to match the way done by the Zephyr CMakeFiles